### PR TITLE
[Docs-2.0] Update to UsingWasabi-Send-Observers section

### PR DESCRIPTION
**Pull request by jesseaam**
Updates to Observers section. Reworded some sentences and removed a statement that said labels were not required.

### CODE REVIEW

File: docs/using-wasabi/Send.md [hunk 1]
```diff
@@ -65,11 +70,14 @@ Wasabi will calculate the checksum and notify you if the provided address is wro
...
+It is a requirement to provide one or more [labels](/using-wasabi/Receive.md#the-importance-of-labeling) each time you initiate a transfer of bitcoin.
+These labels should be the observers of the transaction.
+An observer of a sending transaction is, of course, the receiver of it, as well as any other third party that will find out about it. 
+For example, you should include as a label the payment processor or the bitcoin exchange if you use their services.
 This metadata will be used to build an accurate cluster of observers who know about your coins.
-A label is not required (except in edge cases) when you send an entire coin, as there will be no change left.
+A change address will be automatically created, if needed, following the BIP44 standard. 
+You should avoid change addresses when possible to improve your privacy (see [Change address detection](https://en.bitcoin.it/wiki/Privacy#Change_address_detection)).
+Wasabi Wallet will provide suggestions to improve the privacy of your transaction on the Preview Transaction page.
```
Comment (yahiheb) [inline docs/using-wasabi/Send.md:80]: I don't understand why all of this was added.

**Review by yahiheb — CHANGES_REQUESTED** [review]

File: docs/using-wasabi/Send.md (hunk 1 above)
Comment (jesseaam) [reply docs/using-wasabi/Send.md:78]: You're right - those lines are immaterial  to the Observers section.

File: docs/using-wasabi/Send.md [hunk 2]
```diff
@@ -28,10 +28,15 @@
...
-In the Wasabi wallet `Send` tab, you see a list of all the coins you can spend.
-You can get a coin by first [receiving](/using-wasabi/Receive.md) them from someone else, for example by earning them or exchanging them for fiat currency.
-You can spend one or more coins by selecting them in the `Send` tab, if your payment amount is below the value of the selected coins, then you will receive a [change coin](/using-wasabi/ChangeCoins.md) back.
+A coin is an unspent transaction output (UTXO): a chunk of bitcoin that can be sent in a future transaction.
+Unlike fiat currencies which have fixed denominations, each UTXO contains a variable amount of bitcoin.
+When you want to send some bitcoin you simply enter the amount to send after clicking the `Send` button.
+Wasabi Wallet does the job of automatically selecting the appropriate combination of coins to include as inputs in the transaction.
+Note that there is an option to manually select and send specific coins using [DeveloperFeatures](https://github.com/zkSNACKs/WalletWasabi/blob/master/WalletWasabi.Documentation/DeveloperFeatures.md).
```
[the remark this answers is not in the document]
Comment (jesseaam) [reply docs/using-wasabi/Send.md]: After our discussion today, it was decided that the sentence about manually selecting coins should not currently be included in this section.

File: docs/using-wasabi/Send.md [hunk 3]
```diff
@@ -28,10 +28,11 @@
 
 ## Coins
 
-A coin is an unspent transaction output (UTXO), a chunk of bitcoin which can be sent in a future transaction.
-In the Wasabi wallet `Send` tab, you see a list of all the coins you can spend.
-You can get a coin by first [receiving](/using-wasabi/Receive.md) them from someone else, for example by earning them or exchanging them for fiat currency.
-You can spend one or more coins by selecting them in the `Send` tab, if your payment amount is below the value of the selected coins, then you will receive a [change coin](/using-wasabi/ChangeCoins.md) back.
+A coin is an unspent transaction output (UTXO): a chunk of bitcoin that can be sent in a future transaction.
```
[the remark this answers is not in the document]
Comment (jesseaam) [reply docs/using-wasabi/Send.md:31]: ACK

File: docs/using-wasabi/Send.md [hunk 4]
```diff
@@ -28,10 +28,11 @@
...
+Unlike fiat currencies which have fixed denominations, each UTXO contains a variable amount of bitcoin.
+You can get coins by first [receiving](/using-wasabi/Receive.md) them from someone else.
+When you want to send some bitcoin you simply enter the amount to send after clicking the `Send` button.
```
Comment (yahiheb) [inline docs/using-wasabi/Send.md:34]:
```suggestion
When you want to send some bitcoin you simply enter the amount to send and the address after clicking the `Send` button.
```

**Review by yahiheb — CHANGES_REQUESTED** [review]

**Pull request merged**